test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -21,33 +21,76 @@ def test_dlc_major_version_label(image, region):
     :param region: <str> region where ECR repository holding the image resides
     :return:
     """
-    ecr_client = boto3.client("ecr", region_name=region)
+    labels = _get_labels_from_ecr(image, region)
+    major_version = labels.get("dlc_major_version", None)
 
-    image_repository, image_tag = test_utils.get_repository_and_tag_from_image_uri(image)
-    # Using "acceptedMediaTypes" on the batch_get_image request allows the returned image information to
-    # provide the ECR Image Manifest in the specific format that we need, so that the image LABELS can be found
-    # on the manifest. The default format does not return the image LABELs.
-    response = ecr_client.batch_get_image(
-        repositoryName=image_repository,
-        imageIds=[{"imageTag": image_tag}],
-        acceptedMediaTypes=["application/vnd.docker.distribution.manifest.v1+json"],
-    )
-    if not response.get("images"):
-        raise KeyError(
-            f"Failed to get images through ecr_client.batch_get_image response for image {image_repository}:{image_tag}"
+    assert major_version, f"{image} has no LABEL named 'dlc_major_version'. Please insert label."
+
+    test_utils.LOGGER.info(f"{image} has 'dlc_major_version' = {major_version}")
+
+
+# TODO: Apply to all images when labels are decided upon
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.integration("dlc_labels")
+@pytest.mark.model("N/A")
+def test_dlc_standard_labels(image, region):
+    customer_type_label_prefix = "e3" if test_utils.is_e3_image(image) else "sagemaker"
+    
+    # TODO: Remove skip condition when labels are added for e3 images
+    if customer_type_label_prefix == "e3":
+        pytest.skip("Labels are not currently added to e3 images")
```

Comment:
Should we skip the test or confirm if there is no label attached to the image?

Reply:
Updated the test to ensure e3 labels are not added